tests/ted.py

```diff
@@ -39,6 +39,10 @@
         nargs='*',
         help='tests to run, if not specified all tests will be executed'
     )
+    parser.add_argument(
+        '--device', action='store', default='/dev/dri/renderD128',
+        help='provide device on which to run tests'
```

Comment:
again: what is the default?

Reply:
Default argument is used if it's not provided by user

Reply:
does it appear in the ted.py --help print out?

Reply:
Yes, it does

Reply:
I meant this option appears, but default value doesn't appear. Do I need to add it?

Reply:
yes, it will be helpful. Optimally I would like to see something like the following to be printed out:

`--device  provide device on which to run tests (default: /dev/dri/renderD128)`

Reply:
Done